oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/server/GRPCHandlerRegisterImpl.java

```diff
@@ -33,7 +35,7 @@ public GRPCHandlerRegisterImpl(GRPCServer server) {
     }
 
     @Override public void addHandler(BindableService handler) {
-        server.addHandler(handler);
+        AuthenticationFilter.INSTANCE.build(server, handler);
```

Comment:
I think you should add a method called 

> @Override public void addFilter()

Then in provider, you should add a filter, add(AuthenticationFilter), first. Then calling addHanlder will make those filters active.

The current implementation is working, but the codes look strange.